src/fdk_rdf_parser/parse_functions/reference_data_code.py

```diff
@@ -103,6 +103,17 @@ def extract_reference_frequency(
         return None
 
 
+def extract_reference_dataset_type(
+    graph: Graph, subject: URIRef
+) -> Optional[ReferenceDataCode]:
+    type_ref = graph.value(subject, DCTERMS.type)
+    if type_ref:
+        dct_type = parse_reference_code(graph, type_ref, DC.identifier, SKOS.prefLabel)
+        return filter_reference_data_code(dct_type)
```

Comment:
Hva er forresten grunnen til at vi filtrerer på innholdet i referanse-koden her? Vil det noensinne komme inn/følge med custom referanse-koder, eller vil de alltid komme fra ReferenceData som vi har kontroll på?

Reply:
Tror vi fortsatt har kodelister i reference-data som ikke bruker "code", er kanskje lite nødvendig akkurat her, men beholdt samme metodikk som er brukt ellers her i samme fil. Vi kan heller fikse når vi faktisk tar kontroll på reference-data og standardiserer alt derfra

Reply:
Noen kan feks legge dette til høsting:
`````
<https://dataset.uri>
  a         dcat:Dataset ;
  dct:type  [ ] .

Reply:
som metoden `parse_reference_code` kommer til å lage et objekt for, men alle feltene vil være tomme